### PR TITLE
Adding Hawaiian digi under + Fix digi BloodRed pajamas

### PR DESCRIPTION
**Pull request by rarytech**
## About The Pull Request

Added digi variations for Hawaiian outfits (sprites+flag). I was also asked to fix blood-red pajamas, but creating a PR just for them seemed unnecessary, i think?.

## How This Contributes To The Skyrat Roleplay Experience
More correct sprites

## Proof of Testing

<details>
<summary>

pic

</summary>
  
</details>

## Changelog

:cl:
add: Added digi variations for tactical Hawaiian outfits
fix: fixed digi blood-red pajamas
/:cl:

### CODE REVIEW

File: code/modules/clothing/under/syndicate.dm
Comment (ghost) [inline code/modules/clothing/under/syndicate.dm]: This needs to be done modularly; as-is, this is editing a TG file directly. Bad for a downstream, will cause merge conflicts whenever TG touches this file.

Look under the modular_skyrat folder for somewhere relevant to put this. A maintainer will probably have an idea of where exactly that might be; IIRC there was a modular_overrides file buried in that mess somewhere for all the cases we added variation flags to items.

Comment (vinylspiders) [inline code/modules/clothing/under/syndicate.dm]:
```suggestion
```
What Orion said, please make this addition a modular override. The file is already created: `master_files/code/modules/clothing/under/syndicate.dm`

Just assign this there instead. We avoid editing TG files when possible.

**Review by vinylspiders — CHANGES_REQUESTED** [review]

File: code/modules/clothing/under/syndicate.dm
Comment (rarytech) [reply code/modules/clothing/under/syndicate.dm]: done

Comment (rarytech) [reply code/modules/clothing/under/syndicate.dm]: will be a lesson to me not to do everything in one PR

Comment (rarytech) [reply code/modules/clothing/under/syndicate.dm]: done

**Review by vinylspiders — APPROVED** [review]
Thanks! Much better. I think the sprites look great but @GoldenAlpharex will want to take a look before this gets merged

**Review by GoldenAlpharex — APPROVED** [review]
Looks good to me!

**Pull request merged**
